src/makeGroupsIntoMarkdown.ts

```diff
@@ -25,23 +25,47 @@ const makeGroupsIntoMarkdown = (
   for (const groups of groupGroups) {
     for (const primaryVal of Object.values(groups.primary)) {
       primaryArr.push(`### ${primaryVal.groupTitle}\n`);
+      if (primaryVal.itemType === "PR" || primaryVal.itemType === "Issue") {
```

Comment:
Can we abstract the table formatting into a helper function to preserve readability here?